packages/material-ui/src/Link/Link.js

```diff
@@ -10,21 +10,28 @@ import Typography from '../Typography';
 
 export const styles = {
   /* Styles applied to the root element. */
-  root: {
-    display: 'inline-block',
-  },
+  root: {},
+  /* Styles applied to the root element if `underline="none"` */
   underlineNone: {
     textDecoration: 'none',
   },
+  /* Styles applied to the root element if `underline="hover"` */
   underlineHover: {
     textDecoration: 'none',
     '&:hover': {
       textDecoration: 'underline',
     },
   },
+  /* Styles applied to the root element if `underline="always"` */
   underlineAlways: {
     textDecoration: 'underline',
   },
+  /* Styles applied to the root element if `block={false}` */
+  inline: {
```

Comment:
What do you think of moving this style to the Typography? It's a feature I wish was present when working on onepixel.com.

Reply:
✨ I like it :) I definitely think it will be popular